noxfile.py

```diff
@@ -129,7 +129,7 @@ def docs(session):
     )
 
 
-@nox.session(python=DEFAULT_PYTHON_VERSION)
+@nox.session(python="3.8")
 def docfx(session):
```

Comment:
Could you also pin sphinx version to 4.0.1 and use `gcp-sphinx-docfx-yaml` instead of `sphinx-docfx-yaml`?

Reply:
2b549a1